samples_project/Assets/SampleViewer/Scripts/AutoImport.cs

```diff
@@ -23,8 +23,20 @@ void Handle(PackageRegistrationEventArgs args)
             RenderPipelineAsset URPasset = AssetDatabase.LoadAssetAtPath<RenderPipelineAsset>("Assets/SampleViewer/Resources/SampleGraphicSettings/SampleURPipeline.asset");
             GraphicsSettings.renderPipelineAsset = URPasset;
 #endif
-        reImport();
+        EditorBuildSettingsScene[] scenes = EditorBuildSettings.scenes;
 
+        foreach (EditorBuildSettingsScene i in scenes)
+        {
+            if (i.path.Contains("VR"))
+            {
+#if USE_OPENXR_PACKAGE
+                    i.enabled = true;
+#else
+                    i.enabled = false;
+#endif
+            }
+        }
+            reImport();
```

Comment:
```suggestion
        foreach (EditorBuildSettingsScene setting in scenes)
        {
            if (setting.path.Contains("VR"))
            {
#if USE_OPENXR_PACKAGE
                    setting.enabled = true;
#else
                    setting.enabled = false;
#endif
            }
        }
            reImport();
```

Reply:
```csharp
 if (setting.path.Contains("VR"))
 ```
What is this all applying to?

Reply:
If there is a scene in the Build Settings scene list named VR, it will disable it if OpenXR is not installed. If you then install OpenXR, it will re-enable the scene.

Reply:
I want the logic determining if the VR Sample is enabled or disabled to be based on if the OpenXR package is installed or not, independent of the platform.